src/Components/OpenModelControl.jsx

```diff
@@ -59,11 +83,50 @@ export default function OpenModelControl({fileOpen}) {
  * @param {Function} setIsDialogDisplayed
  * @return {object} React component
  */
-function OpenModelDialog({isDialogDisplayed, setIsDialogDisplayed, fileOpen}) {
+function OpenModelDialog({isDialogDisplayed, setIsDialogDisplayed, fileOpen, orgNamesArr}) {
+  const {isAuthenticated, user} = useAuth0()
+  // const isAuthenticated = true
+  const [selectedOrg, setSelectedOrg] = useState('')
+  const [selectedRepo, setSelectedRepo] = useState('')
+  const [selectedFile, setSelectedFile] = useState('')
+  const [repoNamesArr, setRepoNamesArr] = useState(['...'])
+  const [filesArr, setFilesArr] = useState(['...'])
+  const theme = useTheme()
+  const navigate = useNavigate()
+  const accessToken = useStore((state) => state.accessToken)
+
+
   const openFile = () => {
     fileOpen()
     setIsDialogDisplayed(false)
   }
+
+  const selectOrg = async (org) => {
+    setSelectedOrg(org)
```

Comment:
Would clarify this code to change the react state var from `selectedOrg` to `selectedOrgName` and then this bit would be:

```
const selectOrg = async (orgNdx) => {
  const orgName = orgNamesArr[orgNdx]
  setSelectedOrgName(orgName)
  ...L110:
    repos = await getRepositories(orgName, accessToken)
```
and below, assuming you change the other variables similarly:
```
`/share/v/gh/${orgName}/${repoName}/main/${fileName}`
```

Reply:
I tried it, not so straight forward.
If you know how to do it, please take this branch and make the change.
Thank you!

Reply:
Sorry about that one, I misunderstood what you were suggesting.